code/datums/diseases/advance/symptoms/oxygen.dm

```diff
@@ -29,10 +29,13 @@ Bonus
 	if(prob(SYMPTOM_ACTIVATION_PROB * 5))
 		var/mob/living/M = A.affected_mob
 		switch(A.stage)
+			if(0, 1, 2, 3)
+				M.remove_trait(TRAIT_NOBREATH, DISEASE_TRAIT)
```

Comment:
I don't think this should be a thing either. A virus shoudln't make you completely O2 immune.

Reply:
Currently what self resp is. Although this might fuck with the new crit maybe?